<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>SET VERTICES</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>  Color</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="vertex_color_r">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>35</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="vertex_color_g">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>35</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="vertex_color_b">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>35</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>  Size</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>  Shape</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="vertex_size">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>67</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="vertex_size_decr_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>65</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vertex_size_incr_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>65</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QComboBox" name="vertex_shape">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>95</y>
     <width>148</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Circle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Square</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>128</y>
     <width>200</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>SET EDGES</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="edge_type">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>225</y>
     <width>148</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>Solid</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dashed</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>153</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>  Color</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>  Type</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="edge_color_g">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="edge_color_b">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="edge_width">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>192</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>187</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>  Width</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="edge_width_incr_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="edge_color_r">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="edge_width_decr_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Projection type</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="projection_type">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>360</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>Parallel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Central</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Background color</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="bg_color_g">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="bg_color_b">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="bg_color_r">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="default_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DEFAULT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>560</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>APPLY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>GRID AND AXIS</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="grid_off">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>70</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="grid_on">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>70</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
